phenopacketlab-frontend/src/environments/environment.dev.ts

```diff
@@ -1,5 +1,5 @@
 const URL = 'https://phenopacketlab-dev.jax.org';
-const API_URL = `${URL}/api/v1`;
+const API_URL = `${URL}/api`;
```

Comment:
Let's remember to change all the routing in the load balancer for the backend redirection as api/v1/* was used there.